extension/src/templates/noprettier/_eslintrc.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+  root: true,
+
+  // Rules order is important, please avoid shuffling them
+  extends: [
+    // Base ESLint recommended rules
+    'eslint:recommended',
+
+    // ESLint typescript rules
+    // See https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#usage
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+
+    // `plugin:vue/essential` by default, consider switching to `plugin:vue/strongly-recommended`
+    //  or `plugin:vue/recommended` for stricter rules.
+    // See https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
+    'plugin:vue/essential'
+  ],
+
+  plugins: [
+    // Required to apply rules which need type information
+    '@typescript-eslint',
+    // Required to lint *.vue files
+    // See https://eslint.vuejs.org/user-guide/#why-doesn-t-it-work-on-vue-file
+    'vue'
+    // Prettier has not been included as plugin to avoid performance impact
```

Comment:
This should be removed in the "no-prettier" version

Reply:
ok, let's get a new PR going then.  If we need a new issue to capture the issues we should do so.